com.unity.shadergraph/Editor/Data/Graphs/Texture2DArrayShaderProperty.cs

```diff
@@ -23,7 +56,7 @@ internal Texture2DArrayShaderProperty()
 
         internal override string GetPropertyBlockString()
         {
-            return $"{hideTagString}{modifiableTagString}[NoScaleOffset]{referenceName}(\"{displayName}\", 2DArray) = \"white\" {{}}";
+            return $"{hideTagString}{modifiableTagString}[NoScaleOffset]{referenceName}(\"{displayName}\", 2DArray) = \"\" {{}}";
```

Comment:
Why was white removed?

Reply:
This is invalid for texture arrays and leads to errors. I don't think it was ever tested.